vscode-extension/src/extension.ts

```diff
@@ -69,12 +70,26 @@ export async function activate(context: ExtensionContext) {
     revealOutputChannelOn: RevealOutputChannelOn.Never
   };
 
-  const disposable = new LanguageClient(
+  const client = new LanguageClient(
     'scalameta',
     'Scalameta',
     serverOptions,
     clientOptions
-  ).start();
+  );
 
-  context.subscriptions.push(disposable);
+  const restartServerCommand = commands.registerCommand("scalameta.restartServer", async () => {
+    await exec("jps | grep coursier | awk '{ print $1 }' | xargs kill");
```

Comment:
This might catch other long running coursier processes, I would be careful to run this on client machines. Is there any way for us to get the pid of the process of the server when we start it?

Reply:
Ah, right, it's a bit overkill this way. I may be able to retrieve the server pid from the client. I'll try later

Reply:
done! I had to access a private variable of the language client, but it seems to work reliably. `private` a is compile-time only concept in TypeScript and can be easily bypassed.